nutrition_labels/grant_tagger.py

```diff
@@ -127,7 +136,8 @@ def return_mislabeled_data(self, y_actual, y_pred, X_indices):
         ngram_range=(1,2),
         test_size=0.25,
         irrelevant_sample_seed=4,
-        split_seed= 4
+        split_seed= 4,
```

Comment:
for this bit I'd make a function 'grant_tagger_experiments' (not in the GrantTagger class though) in which you input the vectorizer type and model type, and then do these 3 experiments. otherwise you are copying and pasting quite a bit of code.